### DISCUSSION TITLE
pagination

### DISCUSSION THREAD

**Discussion by DuajDiaz** [DuajDiaz/Travel-Website#57] [Q&A] [answered]
What is pagination? How can pagination be implemented?

**Comment by Bunny77K9** [comment, marked as answer]
Pagination is a simple sequence of pages on a website. These pages are interconnected and have similar content to display to the users.

A simple example is the page selector on an e-commerce site that allows the users to browse through the products present on multiple pages rather than scrolling up and down on one single page.

It can easily be implemented in CSS3 using the following code:

<div class="main_container">
<div class="pagination">
<ul>
<li><a href="#"></a></li>
<li><a href="#"></a></li>
<li class="active"><a href="#"></a></li>
<li><a href="#"></a></li>
</ul>
</div>
</div>
